app/services/company.js

```diff
@@ -21,6 +21,10 @@ class CompanyService extends CrudService {
       error.throwValidationError('Invalid company format.');
     }
 
+    if (company.timetable === undefined) {
```

Comment:
This is not necessary. The field will be validated by Sequelize based on the rules you set.